tests/sav/nitsmells_args1.res

```diff
@@ -0,0 +1,18 @@
+Test ModelBuilder
+*** CODE SMELLS METRICS ***
+--- Code Smells Metrics ---
+-------------------
+Class: Character
+Feature envy :  true
+	Affected method:
+		-quit
```

Comment:
Feature envy: A method accesses the data of another object more than its own data.

Here the `quit` code:

~~~nit
fun quit do
    career = null
end
~~~

I really don't understand the result.